bin/migrations/shared/index.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Shared migration utilities
+ * Barrel export for all shared utilities
```

Comment:
I don't think we should be using barrel files. Let's discontinue the practice. Remove this and add a lint rule against it in this repository.

Reply:
Do we have eslint in this repo? Do we have a CI action that will pass or fail a branch based on the results? Let's do that in addition to the vitest business.